axlearn/common/checkpointer.py

```diff
@@ -315,11 +315,14 @@ class Config(StateStorage.Config):
                 `None` and `1` means no sharding. `-1` means fully shard along data-parallel
                 replicas. `>1` means custom sharding degree (currently not implemented).
             max_concurrent_gb: Max concurrent shards (in GB) to write.
```

Comment:
Should we rename this `max_current_save_gb` to be symmetric?

Reply:
I think we should avoid changing existing config names?

Reply:
It's actually ok to change existing config names. We can do so in a separate PR though.

Reply:
Sure. We can leave it for a future PR.

Reply:
Done.